packages/cli/src/commands/dataMigrate/up.js

```diff
@@ -155,9 +155,11 @@ export const handler = async () => {
 
   try {
     await tasks.run()
-  } finally {
     await db.$disconnect()
     report(counters)
-    process.exit(0)
+  } catch (e) {
+    await db.$disconnect()
+    report(counters)
+    process.exit(e?.exitCode || 1)
```

Comment:
The `finally` block was really elegant, but this get the job done too.

Reply:
I know :(, but process.exit in the catch would mean it wouldn't run!